src/Homepage/ResultBlock.js

```diff
@@ -1,58 +1,76 @@
 
 function removeSafety(entry) {
-  return {
-    ...entry,
-    "Safety Observations": "",
-    "Useful Information": ""
-  };
+    return {
+        ...entry,
+        "Safety Observations": "",
+        "Useful Information": ""
+    };
 }
 
 export default class ResultBlock extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      hover: false
-    };
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            hover: false
+        };
+    }
 
-  onClick() {
-    window.scrollTo({
-      top: 50,
-      behavior: "smooth"
-    });
-    this.props.onClick && this.props.onClick(this.props.result);
-  }
+    onClick() {
+        window.scrollTo({
+            top: 50,
+            behavior: "smooth"
+        });
+        this.props.onClick && this.props.onClick(this.props.result);
+    }
 
-  render() {
-    const { result } = this.props;
-    const entry = result.entries.length ? result.entries[0] : undefined;
-    return (
-      <div
-        onClick={() => this.onClick()}
-        onMouseEnter={() => this.setState({ hover: true })}
-        onMouseLeave={() => this.setState({ hover: false })}
-        className="card my-1"
-        style={{
-          cursor: "pointer",
-          backgroundColor: this.state.hover ? "#eee" : "white"
-        }}
-      >
-        <div className="card-body p-3">
-          <Link
-            to={{ pathname: "/update", state: { item: removeSafety(entry) } }}
-            className="float-right btn btn-sm btn-outline-success text-uppercase"
-            onClick={recordUpdateStore}
-          >
-            Update
-          </Link>
-          <h5 className="card-title m-0 p-0">{result.name}</h5>
-          <ResultEntry entries={result.entries} />
-        </div>
-      </div>
-    );
-  }
+    render() {
+        const {result} = this.props;
+        const entry = result.entries.length ? result.entries[0] : undefined;
+        return (
+            <div
+                onClick={() => this.onClick()}
+                onMouseEnter={() => this.setState({hover: true})}
+                onMouseLeave={() => this.setState({hover: false})}
+                className="card location-card shadow-sm mb-2"
+                style={{
+                    cursor: "pointer",
+                    backgroundColor: this.state.hover ? "#eee" : "white"
+                }}
+            >
+                <div className="card-header d-flex flex-row justify-content-between">
+                    <h6 className="card-title m-0 p-0 d-flex align-self-center">{result.name}</h6>
+                    <div className="flex-fill location-buttons">
+                        <Link
+                            to={{pathname: "/update", state: {item: entry}}}
+                            className="float-right btn btn-sm btn-outline-primary text-uppercase ml-1"
+                            onClick={recordAddInfoToStoreCard}
```

Comment:
`onClick={recordAddInfoToStoreCard}` is incorrect, this should open google maps with directions.